src/tests/issues.targets

```diff
@@ -27,6 +27,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/baseservices/mono/runningmono/*">
             <Issue>This test is to verify we are running mono, and therefore only makes sense on mono.</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/SequentialLayout/ManagedSequential/LargeStructSize_Mono/*">
+            <Issue>This test is designed to run on mono only.</Issue>
```

Comment:
Would it be better to use `SkipOnCoreClr` attribute instead?

Reply:
I tried and hit this bug: https://github.com/dotnet/runtime/issues/105964